superset-frontend/src/dashboard/components/SliceHeader.jsx

```diff
@@ -46,6 +46,8 @@ const propTypes = {
   componentId: PropTypes.string.isRequired,
   filters: PropTypes.object.isRequired,
   addDangerToast: PropTypes.func.isRequired,
+  handleFullSize: PropTypes.func.isRequired,
+  resizeEvent: PropTypes.bool,
```

Comment:
if this is a boolean, then it should be probably named something like `isResizeEvent` or `hasResizeEvent`. I'm not quite sure what makes the most sense here, but some renaming would be great

Reply:
resizeEvent deleted b79ed4fc5348742c102b9e5313669749eb699700

Reply:
looks like it's still in the PR, maybe you missed a commit?

Reply:
My mistake... fixed f1961d31bc239231178755f38d55eb7caa2af035